packages/clay-color-picker/src/__tests__/index.tsx

```diff
@@ -467,15 +489,113 @@ describe('Interactions', () => {
 
 			fireEvent.click(dropdownToggle as HTMLButtonElement, {});
 
-			const blankSplotch = editorGetByTitle(
+			const colorSplotch = editorGetByTitle(
 				'008000'
 			) as HTMLButtonElement;
 
-			fireEvent.click(blankSplotch as HTMLButtonElement, {});
+			fireEvent.click(colorSplotch as HTMLButtonElement, {});
 
-			expect(document.body).toMatchSnapshot();
+			const colorEditorToggle = (
+				document.body as HTMLElement
+			).querySelector('.clay-color-header button');
+
+			fireEvent.click(colorEditorToggle as HTMLButtonElement, {});
 
 			expect(input.value).toBe('008000');
+
+			expect(
+				document.body.querySelectorAll('button[title="dfcaff"]').length
+			).toBe(0);
+
+			expect(editorGetByTestId('rInput').value).toBe('0');
+			expect(editorGetByTestId('bInput').value).toBe('0');
+			expect(editorGetByTestId('gInput').value).toBe('128');
+
+			expect(editorGetByTestId('customHexInput').value).toBe('008000');
+		});
+
+		it('only updates the inputs and the color editor components', () => {
+			const input = editorGetByLabelText(
+				/Color selection is/
+			) as HTMLInputElement;
+
+			fireEvent.change(input, {target: {value: 'DFCAFF'}});
+
+			const colorEditorToggle = (
+				document.body as HTMLElement
+			).querySelector('.clay-color-header button');
+
+			fireEvent.click(colorEditorToggle as HTMLButtonElement, {});
+
+			const rInput = editorGetByTestId('rInput');
+			const bInput = editorGetByTestId('bInput');
+			const gInput = editorGetByTestId('gInput');
+
+			fireEvent.change(rInput, {target: {value: '200'}});
+
+			fireEvent.change(gInput, {target: {value: '200'}});
+
+			fireEvent.change(bInput, {target: {value: '200'}});
+
+			expect(input.value).toBe('c8c8c8');
+
+			expect(editorGetByTestId('customHexInput').value).toBe('C8C8C8');
```

Comment:
As a final verification test here, you can check if there is any splotch with this color.